content/community/steering/index.md

```diff
@@ -52,14 +54,12 @@ Please contact the committee member leading the project if you would like to lea
 * Global Science - Andrew Lonie
 * Small scale servers - Hans-Rudolf Hotz
 
-## Committee Members
+## GGSC Committee Membership
```

Comment:
include Dave as a member?

Reply:
Excellent idea - our external correspondent! He's a treasure!

Reply:
On reflection: Let's check with Dave first, because we will want to put his affiliation so it's up to him?